src/components/Schedule/Schedule.tsx

```diff
@@ -242,6 +931,7 @@ class Schedule extends React.Component<ISentinelProps, ISentinelState> {
                 </a>
               </li>
             </ul>
+            ChronoLogic 2020
```

Comment:
If the year is supposed to always be the current year then you can replace it with `{new Date().getFullYear()}`